README.md

```diff
@@ -1,269 +1,44 @@
 # Ballerina Twitter Connector
-Connects to Twitter using Ballerina.
 
-# Introduction
-The Twitter API can be used to programmatically retrieve and analyze data, as well as engage with the conversation on Twitter.
-This connector provides access to a variety of different resources in Twitter including the following:
-* Tweets
-* Users
- 
-The Twitter API currently consists of two supported versions, as well as different access tiers.
-When user wants to access Twitter through Ballerina Twitter Connector, they are required to register an application. By default, applications can only access public information on Twitter. Certain operations, such as those responsible for sending or receiving Direct Messages and posting a Tweet, require additional permissions from user before access user information. These permissions are not granted by default; you choose on a per-application basis whether to provide this access, and can control all the applications authorized on your account.
+[![Build](https://github.com/ballerina-platform/module-ballerinax-twitter/workflows/CI/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-twitter/actions?query=workflow%3ACI)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-twitter.svg)](https://github.com/ballerina-platform/module-ballerinax-twitter/commits/master)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-## What is Twitter?
-Twitter is a social networking service on which users can post and communicate with status called as "Tweets". Authenticated users can post, like, and retweet tweets.  Main use case is to connect people and allow people to express their thoughts to large set of people.
+[Twitter](https://twitter.com/) is a social media service provided by Twitter.
```

Comment:
Shall we use a rephrased version of the previous description? Twitter is a social networking service on which users can post and communicate with a status called "Tweets".  Authenticated users can post, like, and retweet tweets. The main goal of Twitter is to connect people and to provide a platform for every person to express their thoughts to a large set of people freely.

Reply:
Fixed